packages/grid/src/Grid.tsx

```diff
@@ -1323,14 +1323,34 @@ class Grid extends PureComponent<GridProps, GridState> {
    * @param viewState New state properties to set.
    * @param forceUpdate Whether to force an update.
    */
-  setViewState(viewState: Partial<GridState>, forceUpdate = false): void {
+
+  setViewState(
+    viewState: Partial<GridState>,
+    forceUpdate = false,
+    eventType: WheelEvent | GridKeyboardEvent | null = null
```

Comment:
Just name it `event`, since you're passing a whole event, not just the type.
Also would just mark it optional:
```
eventType?: WheelEvent | GridKeyboardEvent
```

Reply:
You'll also need to handle a `GridMouseEvent` and wire up `GridVerticalScrollBarMouseHandler.onDrag` and `GridHorizontalScrollBarMouseHandler.onDrag` methods to call the appropriate `setViewState` options.

Reply:
I left this part out intentionally. I thought it would be weird for it to stick to the bottom if the user is dragging the scrollbar to a specific position.

Reply:
That makes sense.